packages/aesirx-dma-app/src/containers/ContentPage/ContentForm/ContentFormGeneral/index.jsx

```diff
@@ -158,19 +158,32 @@ const ContentFormGeneral = observer(
         });
       });
 
+      if (!this.isDisableHeadline()) {
```

Comment:
we don't need this validate chị @trucduy10, because we already required headline in here.
![image](https://github.com/aesirxio/dma-app/assets/106291216/73e057a8-cfc1-467e-a125-4ae9215598dd)